CI/flatpak/com.obsproject.Studio.json

```
@@ -44,6 +44,61 @@
     "*.la"
   ],
   "modules": [
+    {
+      "name": "plog",
+      "buildsystem": "cmake-ninja",
+      "config-opts": [
+        "-DPLOG_BUILD_SAMPLES=OFF"
+      ],
+      "sources": [
```

Plog is a headers-only library and is no longer needed so it can be cleanup completely.

```suggestion
      ],
      "cleanup": ["*"],
      "sources": [
```